### ISSUE TITLE
Reversed weather icons

### ISSUE THREAD

**Issue by astroboii47** [musicpro/temps#2]
Love the app - thanks for keeping it alive! Just a small bug - the night and day icons seem to be switched in the forecasts (it shows the moon during the day and the sun at night). I've tried toggling the 24hr/12hr and it doesn't fix it

**Comment by astroboii47** [comment]
For some context, it only seems to flip in Australia, the other places have correct icons

**Comment by musicpro** [comment]
Mhm... are you from Australia? I checked New Zealand and it is the same for me. But from my point of view it makes sense. So checking the weather in Sydney right now (3:40 PM) from Berlin means that it is night time in Sydney (01:41 AM). So the moon will fit, wouldn't it?

If this is not the case I am happy to try some more debugging, just leave me a hint :)

**Comment by astroboii47** [comment]
Yep I'm from AU - I think the icons are main switched in the weekly forecast section: 
So in the 4 day forecast on the bottom, the moon (moon icon or moon with clouds) will show regardless of what time it is. In the next view with the time scrubber, the icons seem jumbled, with the moon appearing at 3pm and the sun at 3am
![Screen Shot 2019-12-04 at 1 50 34 am](https://user-images.githubusercontent.com/58410541/70061499-7c714880-1638-11ea-8754-676096ad6604.png)
![Screen Shot 2019-12-04 at 1 50 18 am](https://user-images.githubusercontent.com/58410541/70061503-7d09df00-1638-11ea-873e-4934049ea7f6.png)
![Screen Shot 2019-12-04 at 1 50 12 am](https://user-images.githubusercontent.com/58410541/70061505-7d09df00-1638-11ea-8515-ce556149c0a4.png)

**Comment by musicpro** [comment]
Alright that really seems off. So thank you for reporting this. I will try to figure out where this is coming from and then of course fix it, if possible :p

**Comment by musicpro** [comment]
Sorry for the late response. Till now I couldn't find a solution for this. For me it seems connected to the openweather response. And I am not entirely sure how to handle this...
Are you using your own API key from openweatehrmap? 
Ah end do you know since when this is happening? I had to make a change according to the timezone stuff, so maybe this is connected...
